docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+---
+title: MSBuild properties for Microsoft.NET.Sdk
+description: Reference for the MSBuild properties that are understood by the .NET Core SDK.
+ms.date: 02/02/2020
+ms.topic: reference
+---
+# MSBuild properties for .NET Core SDK projects
+
+This page describes MSBuild properties for configuring .NET Core projects.
+
+> [!NOTE]
+> This page is a work in progress and does not list all of the useful MSBuild properties for the .NET Core SDK.
+
+## Framework and dependency properties
+
+- [NetStandardImplicitPackageVersion](#netstandardimplicitpackageversion)
+- [PackageReference](#packagereference)
+- [PackageTargetFallback](#packagetargetfallback)
+- [RuntimeIdentifier](#runtimeidentifier)
+- [RuntimeIdentifiers](#runtimeidentifiers)
+- [TargetFramework](#targetframework)
+- [TargetFrameworks](#targetframeworks)
+
+### NetStandardImplicitPackageVersion
+
+Use the `NetStandardImplicitPackageVersion` property when you want to specify a framework version that's lower than the [metapackage](../packages.md#metapackages) version. The project file in the following example targets `netstandard1.3` but uses the 1.6.0 version of `NETStandard.Library`.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+  </PropertyGroup>
+</Project>
+```
+
+### PackageReference
+
+The `PackageReference` property lets you specify a NuGet dependency. For example, you may want to reference a single package instead of a [metapackage](../packages.md#metapackages). The `Include` attribute specifies the package ID. The project file snippet in the following example references the [System.Runtime](https://www.nuget.org/packages/System.Runtime/) package.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  ...
+  <ItemGroup>
+    <PackageReference Include="System.Runtime" Version="4.3.0" />
+  </ItemGroup>
+</Project>
+```
+
+For more information, see [Package references in project files](/nuget/consume-packages/package-references-in-project-files).
+
+#### Attributes
+
+```xml
+<PackageReference Include="<package-id>" Version="" IncludeAssets="" ExcludeAssets="" PrivateAssets="" />
```

Comment:
Missing the `SkipGetTargetFrameworkProperties` attribute which affects restore compatibility checks.

Also missing the `PublishProfile, which can be used to specify a pubxml file in the case of `wapproj` project types.

Reply:
Thanks, I logged an [issue](https://github.com/NuGet/docs.microsoft.com-nuget/issues/1890) about these attributes in the NuGet docs since I decided not to document PackageReference here, for now.